### ISSUE TITLE
gce --quit-soon does not have expected behavior

### ISSUE THREAD

**Issue by tahorst** [CovertLab/borealis#6]
Running `gce --quit-soon <name>` does not delete the VM after a firework has completed in some cases.  I would expect a worker to check for the metadata status after each firework completes but it looks like `rapidfire` can launch many fireworks before returning for the metadata check here:
https://github.com/CovertLab/borealis/blob/d24b9723d317893821c56c357efee5b75a2ea810/borealis/fireworker.py#L185-L208

I think the arg `nlaunches=1` should be passed to `rapidfire` to exit after launching only one firework so we can check for the quit metadata.  I think `rapidfire` will launch as many rockets that are waiting as it can since it looks like it skips the loop check if more fireworks are ready.
https://github.com/materialsproject/fireworks/blob/6cb2a66d35239611ec2a1ccb807be38976198a0b/fireworks/core/rocket_launcher.py#L107-L126

Is the expectation to check for the metadata after each firework or to let `rapidfire` launch as many as it wants before checking?

**Comment by 1fish2** [comment]
Good idea. It looks like it should set `nlaunches=1` in addition to `max_loops=1`.
